androidHyperskillApp/src/main/java/org/hyperskill/app/android/step_practice/view/fragment/StepPracticeFragment.kt

```diff
@@ -63,11 +63,16 @@ class StepPracticeFragment : Fragment(R.layout.fragment_step_practice), StepComp
             }
             stepQuizToolbarTitle.text = step.title
         }
-        viewBinding.stepPracticeCompletion.text = resourceProvider.getString(
-            SharedResources.strings.step_quiz_stats_text,
-            step.solvedBy.toString(),
-            DateFormatter.formatTimeDistance(step.millisSinceLastCompleted)
-        )
+        with(viewBinding.stepPracticeCompletion) {
+            text = step.millisSinceLastCompleted?.let {
```

Comment:
I think in Android it's also better to use `StepQuizStatsTextMapper.getFormattedStepQuizStats`, like in iOS